src/SFA.DAS.ReferenceData.PublicSectorOrgs.WebJob/Updater/IJsonManager.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.WindowsAzure.Storage;
+using Newtonsoft.Json;
+using SFA.DAS.NLog.Logger;
+using SFA.DAS.ReferenceData.Domain.Models;
+
+namespace SFA.DAS.ReferenceData.PublicSectorOrgs.WebJob.Updater
+{
+    public interface IJsonManager
```

Comment:
Could we have a separate file for interfaces, please.

Reply:
oops, meant to do that, will sort now